bin/cml/runner/start.js

```diff
@@ -0,0 +1,582 @@
+const { join } = require('path');
```

Comment:
still prefer `cml runner launch` to `cml runner start`.

Launching is a superset. It means starting/ignition, configuring/setting up, and taking off. It also removes overloaded/conflicting definition of the word `start` in `cml tensorboard start`. [Overloading (homonyms) are bad for UX](https://tldr.cdcl.ml/arc42-documentation-principles).

Reply:
> _~Overloading (homonyms) are bad for UX~_
> [_Glossary: domain/technical terms; avoid synonyms & homonyms to maximise identical understanding_](https://tldr.cdcl.ml/arc42-documentation-principles)

Similarly with synonyms, as per the same page: currently we're in a situation where all the `cml` commands (nouns) have a single subcommand (verb) that performs an action that can be generalized as a `create` action.

While having `create` everywhere may look confusing and repetitive, using unique words to name the same kind of [positive] action isn't any better, to my mind.

Reply:
> having `create` everywhere may look confusing and repetitive

yes that's a con

> using unique words to name the same kind of [positive] action isn't any better

a) not a con and b) I disagree, it's not at all "the same kind of action"

description | cml runner rerun | cml runner restart
-|-|-
matches term used by GitHub[^1][^2] | :heavy_check_mark: | :x:
matches term used by GitLab[^gl] | :x: | :x:
matches term used by Bitbucket[^bb] | :heavy_check_mark: | :x:
accurately describes behaviour[^desc] | :heavy_check_mark: | :x:

to me, "rerun" is in the same wordcloud as "stop/halt/kill/retry/redo" while "restart" goes with "play/pause/resume"

[^1]: https://cli.github.com/manual/gh_run_rerun
[^2]: https://cli.github.com/manual/gh_workflow_run
[^gl]: https://forum.gitlab.com/t/retry-a-job-multiple-times/58245
[^bb]: https://bitbucket.org/blog/rerun-failed-pipeline-steps
[^desc]: redoing everything from the top of the workflow, i.e. can NOT be mistakenly interpreted to mean "resuming from where it was interrupted"

Reply:
@casperdcl, can you please enumerate all the “casperified” names for every verb?

E.g.

| Current verb | New verb |
| ------------ | -------- |
| `cml workflow restart` | `cml workflow rerun` |
| `cml pr created | `cml pr discombobulate` |

Reply:
current | proposed (casper)
-|-
workflow restart | workflow rerun
runner start | runner launch

`{pr,comment,check} create` (current) are fine.

Reply:
You're missing `tensorboard`

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/a06918c954aa9e3be5f6bca9e271ff168a409b8b

Reply:
Please open a new thread if you want to rename any other verb

Reply:
With regard to `restart` versus `rerun`... https://github.com/iterative/cml/pull/1073/commits/6429c18a33ac873de58351a88d40ebfcd0b5347f was pushed 13 days ago. 🤔